pavelib/assets.py

```diff
@@ -692,30 +692,9 @@ def collect_assets(systems, settings, **kwargs):
     `settings` is the Django settings module to use.
     `**kwargs` include arguments for using a log directory for collectstatic output. Defaults to /dev/null.
     """
-    ignore_patterns = [
-        # Karma test related files...
-        "fixtures",
-        "karma_*.js",
-        "spec",
-        "spec_helpers",
-        "spec-helpers",
-        "xmodule_js",  # symlink for tests
-
-        # Geo-IP data, only accessed in Python
-        "geoip",
-
-        # We compile these out, don't need the source files in staticfiles
-        "sass",
-    ]
-
-    ignore_args = " ".join(
-        f'--ignore "{pattern}"' for pattern in ignore_patterns
-    )
-
     for sys in systems:
         collectstatic_stdout_str = _collect_assets_cmd(sys, **kwargs)
-        sh(django_cmd(sys, settings, "collectstatic {ignore_args} --noinput {logfile_str}".format(
-            ignore_args=ignore_args,
+        sh(django_cmd(sys, settings, "collectstatic --noinput {logfile_str}".format(
```

Comment:
This is not relevant, but I think this is a good opportunity to use f-string since they are more concise and readable

Reply:
I agree that f-strings are better, but in this case I'd prefer to isolate my refactoring by making the most minimal possible update to this file. For what it's worth, I plan to completely delete this file as part of the larger effort.